File: Php/Human.php
<?php

class Human // Membuat class Human (parent class)
{
    /* Membuat atribut berjenis protected */
    protected $identity; // Untuk menyimpan nama
    protected $name; // Untuk menyimpan NIM (ID number)
    protected $gender; // Untuk menyimpan jurusan

    /* Konstruktor */
    public function __construct($identity = 0, $name = '', $gender = '') // Membuat konstruktor dengan isian dari parameter
    {
        $this->identity = $identity;         // Meng-assign atribut identity dengan nilai dari parameter identity
        $this->name = $name; // Meng-assign atribut name dengan nilai dari parameter name
        $this->gender = $gender;       // Meng-assign atribut gender dengan nilai dari parameter gender
    }

    /* Setter dan Getter */
    public function setIdentity($identity) // Setter untuk atribut identity
    {
        $this->identity = $identity; // Meng-assign atribut identity dengan nilai dari parameter identity
    }

    public function setName($name) // Setter untuk atribut name
    {
        $this->name = $name; // Meng-assign atribut name dengan nilai dari parameter name
    }

    public function setGender($gender) // Setter untuk atribut gender
    {
        $this->gender = $gender; // Meng-assign atribut gender dengan nilai dari parameter gender
    }

    public function getIdentity() // Getter untuk atribut identity
    {
        return $this->identity; // Me-return nilai dari atribut identity
    }

    public function getName() // Getter untuk atribut name
    {
        return $this->name; // Me-return nilai dari atribut name
    }

    public function getGender() // Getter untuk atribut gender
    {
        return $this->gender; // Me-return nilai dari atribut gender
    }

    /* Destuktor */
    public function __destruct()
    {
    }
}

File: Python/human.py
class Human: # Deklarasi class Human (parent class)
    _identity = 0 # Untuk menyimpan NIK (Identity Number) mahasiswa, atribut berjenis protected
    _name = "" # Untuk menyimpan nama mahasiswa, atribut berjenis protected
    _gender = '' # Untuk menyimpan jenis kelamin mahasiswa, atribut berjenis protected

    def __init__(self, identity = 0, name = "", gender = ''): # Membuat konstruktor
        self._identity = identity # Menginisialisasi atribut _identity
        self._name = name # Menginisialisasi atribut _name
        self._gender = gender # Menginisialisasi atribut _gender

    def setIdentity(self, identity): # Setter untuk atribut _identity
        self._identity = identity # Menginisialisasi atribut _identity

    def setName(self, name): # Setter untuk atribut _name
        self._name = name # Menginisialisasi atribut _name

    def setGender(self, gender): # Setter untuk atribut _gender
        self._gender = gender # Menginisialisasi atribut _gender

    def getIdentity(self): # Getter untuk atribut _identity
        return self._identity # Mengembalikan nilai atribut _identity

    def getName(self): # Getter untuk atribut _name
        return self._name # Mengembalikan nilai atribut _name

    def getGender(self): # Getter untuk atribut _gender
        return self._gender # Mengembalikan nilai atribut _gender
